CHANGELOG.md

```diff
@@ -4,6 +4,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [3.27.0] - 2025-11-23
+
+### Changed
+* Use the exit code `130` on `SIGINT` by @andrewnicols in [#1686](https://github.com/Behat/Behat/pull/1686)
+* Prepare the pretty printer for future changes in the Gherkin parser by @stof in [#1690](https://github.com/Behat/Behat/pull/1690)
```

Comment:
Small thing but I'd probably put #1690 in Internal as it shouldn't have any externally visible impact

Reply:
good idea